build.gradle.kts

```diff
@@ -17,14 +17,14 @@ repositories {
 
 buildscript {
     dependencies {
-        val smithyVersion = "1.21.0"
+        val smithyVersion = "1.25.2"
         classpath("software.amazon.smithy:smithy-openapi:$smithyVersion")
         classpath("software.amazon.smithy:smithy-aws-traits:$smithyVersion")
         classpath("software.amazon.smithy:smithy-cli:$smithyVersion")
     }
 }
 
 dependencies {
-    implementation("software.amazon.smithy:smithy-model:1.21.0")
-    implementation("software.amazon.smithy:smithy-aws-traits:1.21.0")
+    implementation("software.amazon.smithy:smithy-model:1.25.2")
```

Comment:
`1.26.0` seems to be the latest version. 

https://awslabs.github.io/smithy/1.0/guides/building-models/gradle-plugin.html#:~:text=gradle.kts%3A-,Kotlin,-buildscript%20%7B%0A%20%20%20%20//%20...

Reply:
Yeah the latest version came in between when I started working on this to me raising the PR 🤦🏼‍♀️ Let me update it.